packages/react-icons-mdl2/src/utils/SvgIcon.scss

```diff
@@ -10,4 +10,8 @@
   height: 100%;
   fill: currentColor;
   vertical-align: top;
+  // Temporary forcing color to adjust in HCM until browsers do this properly
+  // https://github.com/microsoft/fluentui/issues/22410
+  forced-color-adjust: auto;
+  --ms-forced-color-adjust: auto;
```

Comment:
I don't think we need the `--ms` prefix, since I'm pretty sure `forced-color` CSS props are only supported on browsers that don't need `--ms` anyway.

Reply:
```suggestion
```